### DISCUSSION TITLE
Delete model file

### DISCUSSION THREAD

**Discussion by carlinogianluca** [Anjok07/ultimatevocalremovergui#1458] [Q&A]
Hi, how can I delete one model file to free space?

**Comment by tmhanif2404** [comment]
They are at "C:\Users\Username\AppData\Local\Programs\Ultimate Vocal Remover\models"

or

You can press wrench icon [Setting] > [Open Application Directory]

![image](https://github.com/user-attachments/assets/a04423be-8755-4d44-9350-398278429165)
